specification/iotoperations/IoTOperations.Management/main.tsp

```diff
@@ -0,0 +1,117 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+import "./models/base.tsp";
+import "./models/auth.tsp";
+import "./models/instance.tsp";
+import "./models/broker/brokers.tsp";
+import "./models/broker/authentications.tsp";
+import "./models/broker/authorizations.tsp";
+import "./models/broker/listeners.tsp";
+import "./models/dataflows/dataflowEndpoints.tsp";
+import "./models/dataflows/dataflowProfiles.tsp";
+import "./models/dataflows/dataflows.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using TypeSpec.OpenAPI;
+using Azure.Core;
+using Azure.ResourceManager;
+
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+@armProviderNamespace
+@service({
+  title: "Microsoft.IoTOperations management service.",
+})
+@doc("Microsoft.IoTOperations Resource Provider management API.")
+@versioned(Versions)
+namespace Microsoft.IoTOperations;
+
+/** Api versions */
+enum Versions {
```

Comment:
>Versions [](http://example.com/codeflow?start=5&length=8)

you added also folders for 2024-06-01-preview version. Is this version used?

Reply:
removed